sdk/lib/rtl/amd64/unwind.c

```diff
@@ -699,6 +711,11 @@ RtlpUnwindInternal(
     /* Start looping */
     while (TRUE)
     {
+        if (!RtlpIsStackPointerValid(UnwindContext.Rsp, StackLow, StackHigh))
```

Comment:
Note: It seems the `EstablisherFrame` StackLow/High/"alignment" checks done below (new line 750) could also benefit from this function. (Maybe renaming this new function into something like "RtlpIsPointerInRange" or something...)

Reply:
Yes, and that will eventually be fixed. This whole function is still WIP.